backend/homework/4/src/main/java/org/example/question2/Main.java

```diff
@@ -0,0 +1,30 @@
+package org.example.question2;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Main {
+
+    private static final Logger logger = LoggerFactory.getLogger(Main.class);
+
+    public static void main(String[] args) {
+        TicketReservation reservationSystem = new TicketReservation();
+
+        boolean bookingResult = reservationSystem.bookFlight("Tony", "Soprano", 30, "Male", "Economy", "1");
+
+        if (bookingResult) {
+            logger.info("Booking successful!");
+        } else {
+            logger.error("Booking failed. The confirmed list and waiting list are full.");
```

Comment:
Its okay but try to use exceptions for throwing the same, as there can be some other logic for failing also.